firestore/google/cloud/firestore_v1beta1/_helpers.py

```diff
@@ -897,7 +893,7 @@ def pb_for_delete(document_path, option):
     """
     write_pb = write_pb2.Write(delete=document_path)
     if option is not None:
-        option.modify_write(write_pb, no_create_msg=NO_CREATE_ON_DELETE)
+        option.modify_write(write_pb)
```

Comment:
@chemelnucfin Why was this removed but not replaced with something equivalent?

Reply:
That message is only good for create_if_missing.